packages/styles/scss/patterns/sub-patterns/content-block/_content-block.scss

```diff
@@ -86,6 +72,70 @@
   }
 }
 
+@mixin content-block-inverse {
```

Comment:
@andysherman2121 is this a modifier to the main `content-block`? I'm wondering if you should just target what is changing rather than duplicate the other mixin entirely?